server/src/api/site.rs

```diff
@@ -614,6 +635,10 @@ impl Perform for Oper<TransferSite> {
     };
 
     let user_id = claims.id;
+    let mut user = blocking(pool, move |conn| User_::read(conn, user_id)).await??;
+    user.password_encrypted = "".to_string();
+    user.private_key = None;
+    user.public_key = None;
```

Comment:
Maybe add a function like `User_::read_safe()`.

Reply:
I'll add a TODO in here for that. But you're right, we do need to make another rust type that excludes these.